build-gluster-org/jobs/user-update.yml

```diff
@@ -0,0 +1,36 @@
+- job:
+    name: update-admin-list
+    node: 
+    description: This job updates the jenkins job's admin-list with the members of gluster-maintainers 
+    project-type: freestyle
+    concurrent: true
+
+    scm:
+    - git:
+        url: git@github.com:gluster/build-jobs.git
+        credentials-id: github-credentials
+        branches:
+        - remotes/origin/master
+        basedir: "build-jobs"
+
+    builders:
+    - shell: |
+        cat << EOF > requirements.txt
+        gitdb==4.0.5
+        GitPython==3.1.9
+        EOF
+        pip install -r requirements.txt
```

Comment:
The script do not seems to use gitdb nor gitpython, afaik. Is there something missing, or is it a left over ?